src/helpers/spaces.ts

```diff
@@ -89,15 +89,16 @@ function mapSpaces() {
 
 async function loadSpaces() {
   const query =
-    'SELECT id, settings, flagged, verified FROM spaces WHERE deleted = 0 ORDER BY id ASC';
```

Comment:
we need to add this field to the `proposals`, `proposal`, and `votes` methods too, like here https://github.com/snapshot-labs/snapshot-hub/pull/709/